charts/seed-bootstrap/charts/elastic-kibana-curator/templates/kibana/kibana-objects-registration-config.yaml

```diff
@@ -63,12 +73,33 @@ data:
         fi
     }
 
+    function create_saved_objects {
+      HTTP_RESPONSE=$(curl --silent \
+         -H "Content-Type: application/json" \
+         -H "kbn-xsrf: true" \
+         --write-out "HTTPSTATUS:%{http_code}" \
+         -X POST $KIBANA_HOST/api/kibana/dashboards/import?exclude=index-pattern \
```

Comment:
As far as I see, we don't have an index-pattern in dashboard.json. Maybe it is a left-over and we can remove the exclude query param?